docroot/modules/custom/sitenow_articles/sitenow_articles.libraries.yml

```diff
@@ -0,0 +1,4 @@
+gallery-jump-link:
+  css:
+    theme:
+      css/gallery-jump-link.css: {}
```

Comment:
I like the idea of this but it might be more useful to move this to https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/content/node--type--article.scss so we can keep all of the article-related css together?

Reply:
I think I could go either way on it. The block it targets is defined in the module, but keeping all article-related css together makes sense to me.

Reply:
I can refactor this

Reply:
That sounds good to me. Thanks Alan!

Reply:
This has been resolved correct?

Reply:
Yep! Should be good to go here.